tests/test_elements.py

```diff
@@ -0,0 +1,105 @@
+import os
+import pytest
+from playwright.async_api import async_playwright
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    "html_file, expected_tag_to_xpath",
+    [
+        ("test_html/text_only.html", {0: "//html/body/h1", 1: "//html/body/p"}),
+        (
+            "test_html/hyperlink_only.html",
+            {
+                0: '//html/body/p[1]/a[@id="link1"]',
+                1: '//html/body/p[2]/a[@id="link2"]',
+                2: '//html/body/p[3]/a[@id="link3"]',
+                3: '//html/body/p[4]/a[@id="link4"]',
+                4: '//html/body/p[5]/a[@id="link5"]',
```

Comment:
Should just have 1 in these tests and combination should have like 3 things. Best to keep these as simple and small as possible